### PR TITLE
Fixed error

### LINKED ISSUE #161: Fix bug with pipeline.py
As a developer, when dealing with the meteo API, I want to extract relevant specific information about sunrise schedules to a local format using API requests so that the data can then be manipulated and processed using python.

- Rewrite pipeline.py so that it conforms to lambda convention

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #161 

I have Rewritten pipeline.py for the openmeteo api to fix an error with the docker image

### CODE REVIEW

File: weekly-openmeteo/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 requests
-psycopg2
+psycopg2-binary
```
Comment (MurasatoNaoya) [inline weekly-openmeteo/requirements.txt:2]: Important change, running the container without the psycopg in binary format can lead to issues.

File: weekly-openmeteo/pipeline.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+'''Orchestrates weekly ETL pipeline'''
+#pylint: disable=W0612,W0613,E0606
+import logging
+from dotenv import load_dotenv
+from extract import extract
+from transform import transform
+from load import load_data
+
+def lambda_handler(event, context):
```
Comment (MurasatoNaoya) [inline weekly-openmeteo/pipeline.py:9]: The lambda function runs the pipeline with clear logging. LGTM.

**Review by MurasatoNaoya — APPROVED** [review]
LGTM.

File: weekly-openmeteo/Dockerfile
Comment (olliejth) [inline weekly-openmeteo/Dockerfile:1]: Important addition

**Review by olliejth — APPROVED** [review]
Well implemented logging, the same can be done for the astronomy script.

**Pull request merged**